src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,56 @@
+import { getArticle } from "../api";
+import CommentsExtension from "./CommentsExtension";
+import CommentForm from "./CommentForm";
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+const ArticlePage = () => {
+  const { article_id } = useParams();
+  const query = `/${article_id}`;
+  const [article, setArticle] = useState([]);
+  const [showComments, setShowComments] = useState(false);
+  const [addVotes, setAddVotes] = useState(false);
+
+  useEffect(() => {
+    getArticle(query).then((response) => {
+      setArticle(response);
+    });
+  }, [query]);
+
+  const handleShowComments = () => {
+    setShowComments((prevState) => !prevState);
+  };
+
+  const handleHideComments = () => {
```

Comment:
Does this function need to be here or could the handleShowComments be used instead since its flexible enough to flip the state either way?